Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org="ballerinax"
 name="mssql.driver"
-version="1.1.0"
+version="1.2.0"
 authors=["Ballerina"]
```

Comment:
Let's add distribution property and remove observability configs

Reply:
Fixed in 4cb252a